website/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "website",
+  "name": "@microsoft/autogen-website",
```

Comment:
Considering this package is not published can we just remove the name and version?

Reply:
Agree, this is package is not published. Also suggest instead of @microsoft/autogen-website just autogen-website